test/test_view.c

```diff
@@ -29,6 +29,7 @@ DEALINGS IN THE SOFTWARE.  */
 #include <unistd.h>
 #include <stdlib.h>
 #include <string.h>
+#include <getopt.h>
```

Comment:
No problem.  Including extra header files isn't a major problem and as long as it helps one platform out there I have no issue with it at all.  If I had then I wouldn't have merged it as was.  It was simply my curiosity and to consider whether there are better ways for us to be doing automated testing on a wider range of platforms.